### PR TITLE
chore: update folder structure and readme

### PR DESCRIPTION
**Pull request by Guillaume-sousa**
- Update folder structure to have 2 installation options at the root (helm or manifests)
- Update readme with pointers to the 2 options

### CODE REVIEW

File: helm-chart/README.md [hunk 1]
```diff
@@ -1,7 +1,9 @@
 # Install with Helm
 
 ## Prerequisites
-* Helm: https://helm.sh/docs/intro/install/
+* [Kubernetes Cluster](https://kubernetes.io/docs/setup/production-environment/tools/kubeadm/create-cluster-kubeadm/)
```
Comment (MyleneSimon) [inline helm-chart/README.md]: I would link to the more general "Kubernetes setup options" page instead https://kubernetes.io/docs/setup/

**Review by MyleneSimon — APPROVED** [review]

File: manifests/README.md [hunk 2]
```diff
@@ -0,0 +1,344 @@
+# Install with manifests
+
+Follow the instructions below to deploy the CDCS in a Kubernetes cluster
+using YAML manifests.
+
+## Prerequisites
+* [Kubernetes Cluster](https://kubernetes.io/docs/setup/production-environment/tools/kubeadm/create-cluster-kubeadm/)
```
Comment (pdessauw) [inline manifests/README.md]: same comment as Mylene here

File: helm-chart/README.md [hunk 3]
```diff
@@ -1,7 +1,9 @@
...
+* [Helm](https://helm.sh/docs/intro/install/)
+* [Ingress Nginx](https://kubernetes.github.io/ingress-nginx/deploy/)
 * PV provisioner support in the underlying infrastructure (or see the [Volumes](#volumes) section to use existing claims)
```
Comment (pdessauw) [inline helm-chart/README.md:7]: Point directly to the volume part in the manifest README

File: helm-chart/README.md [hunk 4]
```diff
@@ -120,10 +127,10 @@ auth:
 
 ## Volumes
```
Comment (pdessauw) [inline helm-chart/README.md:137]: move this part in the volume section of the manifests README

File: helm-chart/README.md [hunk 5]
```diff
@@ -40,6 +42,11 @@ Override only specific fields of the `values.yaml` file, with the `--set` option
 helm install -n mdcs-test --create-namespace mdcs-helm-test . --set cdcs.imagePullPolicy=IfNotPresent
 ```
 
+The CDCS container can be configured using environment variables. 
+The complete list of environment variables can be found in the
+[customized deployment](https://github.com/usnistgov/cdcs-docker?tab=readme-ov-file#1-customize-the-deployment)
+section of the CDCS Docker documentation.
+
```
Comment (pdessauw) [inline helm-chart/README.md:58]: Can we add more info on how to create these variables in this case and make sure they are used by the app?

**Review by pdessauw — CHANGES_REQUESTED** [review]

**Review by pdessauw — APPROVED** [review]
lgtm!

**Pull request merged**